packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -488,15 +490,20 @@ const disableVmHighAvailability = async (xapi: Xapi, vm: Vm) => {
 //
 // Attributes of created VMs:
 //
-// - all snapshots attributes (see above)
+// - [offline backup mode disabled] all snapshots attributes (see above)
+// - [offline backup mode enabled] `other_config`:
+//    - `xo:backup:datetime` = new Date()
```

Comment:
According to the documentation, `snapshot.snapshot_time` can be a date or a time but after looking at the different values that can takes this property using the `xen-api` CLI, i see that it's always a date.

Reply:
My bad it's not date or time but its date with time :+1:

Reply:
What is the format?

Is it the same as `String(Date.now())`?

Reply:
I changed it to `new Date()`.

Reply:
I need to use a UTC format to do this i'll use the `'d3-time-format'` precisely the function `utcFormat` to have this format `YmdTH:M:SZ`.

Reply:
This should be documented.

I regret not having documented the previous format.

Reply:
I documented the format of `snapshot.snapshot_time`.